src/hotspot/share/runtime/statSampler.cpp

```diff
@@ -215,76 +214,58 @@ const char* StatSampler::get_system_property(const char* name, TRAPS) {
  * creates other property based instrumentation conditionally.
  */
 
-// stable interface, supported counters
-static const char* property_counters_ss[] = {
+// stable interface, supported counters in the JAVA_PROPERTY name space
```

Comment:
```
 * The list of System Properties that have corresponding PerfData
 * string instrumentation created by retrieving the named property's
 * value from System.getProperty() and unconditionally creating a
 * PerfStringConstant object initialized to the retrieved value. This
 * is not an exhaustive list of Java properties with corresponding string
 * instrumentation as the create_system_property_instrumentation() method
 * creates other property based instrumentation conditionally.
```
I found the above comment unreadable. Is it possible to clarify it?

Reply:
I reworked it a bit to get rid of the static array and keep comments more coherent. I think it reads a bit better now.